src/main/java/org/alliancegenome/curation_api/interfaces/crud/VocabularyCrudInterface.java

```diff
@@ -28,4 +28,9 @@
     @JsonView(View.VocabularyTermView.class)
     ObjectListResponse<VocabularyTerm> getTerm(@PathParam("id") Long id);
     
+    @GET
+    @Path("/findBy/{name}")
+    @JsonView(View.FieldsAndLists.class)
+    public ObjectResponse<Vocabulary> get(@PathParam("name") String name);
```

Comment:
I think is would be good to change the method name from `get` to `findByName` it just makes it more clear.

Reply:
Done